apps/backend/src/users/users.controller.ts

```diff
@@ -7,7 +13,10 @@ export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
   @Get()
-  getAllUsers() {
-    return this.usersService.findAll();
+  getAllMembers(
```

Comment:
:thought_balloon: Just thinking out loud but I have a feeling we'll want this route to be a more generic route that can be used to retrieve users of an arbitrary list of statuses instead of having separate routes to get different subsets of users

Reply:
For sure, we can go back to this and change it to be more generic in the future (unless you want us to refactor it now)

Reply:
Yeah let's change it in a future PR for sure